web/modules/custom/joinup_core/joinup_core.routing.yml

```diff
@@ -54,3 +54,13 @@ joinup_core.id_redirect:
     _controller: '\Drupal\joinup_core\Controller\IdRedirect::redirectToRdfEntity'
   requirements:
     _access: 'TRUE'
+
+joinup_core.go_to_rdf_entity_page:
+  path: '/rdf'
```

Comment:
Can we have it as a bit more unique url? Rdf is a very common thing to use. I know it is project specific as it is in joinup_core but since this is a distribution, I think it's better to not use this keyword for the url. Maybe rdf-redirect or something similar.

Reply:
The idea was to provide a handy tool, an "easy to type" thing. If we put there a long URL will not help. Right now, on our project, this URL is not used. If it will be in the future, we can change it.